src/Bicep.Core.IntegrationTests/DeployTimeConstantTests.cs

```diff
@@ -73,7 +77,7 @@ public void DtcValidation_EntireResourceOrModuleAccessAtInvalidLocations_Produce
 
 resource subnet2 'Microsoft.Network/virtualNetworks/subnets@2022-07-01' = {
   name: 'subnet2'
-  parent: vnet 
```

Comment:
This was automatically done by my IDE. I'm thinking I should turn this off given that some of these tests might expect trailing whitespace.